tester/src/accounts/account.rs

```diff
@@ -0,0 +1,22 @@
+// Copyright 2022 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+use serde_json::Value;
+
+use crate::{
+    accounts::{amount::process_amount, outputs::process_outputs},
+    context::Context,
+    error::Error,
+};
+
+pub async fn process_account<'a>(context: &Context<'a>, account: &Value) -> Result<(), Error> {
+    let created_account = context.account_manager.create_account().finish().await?;
+
+    if account.is_u64() {
+        process_amount(context, &created_account, account.as_u64().unwrap()).await?;
+    } else if account.is_array() {
+        process_outputs(context, &created_account, account.as_array().unwrap()).await?;
```

Comment:
Why not just this one here? The extra function with another name is not really clear here
```suggestion
    for output in account.as_array().unwrap() {
        if output.is_u64() {
            process_amount(context, account, output.as_u64().unwrap()).await?;
        }
    }
```
Also the whole naming, why is it called `account` if it's the amount?

Reply:
Because there will be more types of output in the future, amount is just one option to quickly create a basic output with this value

Reply:
And same reason for the naming, account will not always be an amount

Reply:
But then let's call it `output` or why `account`?

Reply:
Because it's an array of outputs

Reply:
Ok then why not call it `outputs`? :carlos:

Reply:
I'm afraid that if we call the field `outputs`, it's not easy to understand that it's actually creating an account per index

Reply:
And accounts is more flexible, we may want to set more things than outputs in the future

Reply:
Huh, where is it creating a new account?

Reply:
L13

Reply:
Yes but that's before and only a single account independent of one or more

Reply:
I can rename it but it's going to be obsolete very soon as we'll need to actually have an account object instead of what we have now

Reply:
Changing anyway

Reply:
Done :carlos: